misc/proxy/haproxy/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
nb: the .sh extension on the filename is unnecessary/redundant. Ican just be "build" as it's an executable file, and it doesn't matter if internally it's shell or python, etc...!